### PR TITLE
[PCX-3760] Add option to manually upload to Browserstack

### PR DESCRIPTION
**Pull request by Sorix**
> https://optile.atlassian.net/browse/PCX-3760

## Overview
Add option to manually upload to Browserstack through GitHub Actions.

## Checklist
_Not applicable_

### CODE REVIEW

File: .github/workflows/publish_browserstack.yml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 name: Publish to BrowserStack
 
 on:
+  workflow_call:
```
Comment (caiomello) [inline .github/workflows/publish_browserstack.yml:4]: It's supposed to be `workflow_dispatch`, no?

Comment (Sorix) [reply .github/workflows/publish_browserstack.yml]: Thank you, changed in cd3671b7fc4626a5834315c75ed2bfc935e9d8f6

File: .github/workflows/publish_browserstack.yml [hunk 2]
```diff
@@ -1,6 +1,8 @@
-name: Publish to BrowserStack
+name: 'ACTION: Publish to BrowserStack'
```
Comment (caiomello) [inline .github/workflows/publish_browserstack.yml:1]: Do we really need to use a separate workflow and name?

**Review by caiomello — APPROVED** [review]

File: .github/workflows/publish_browserstack.yml (hunk 2 above)
Comment (Sorix) [reply .github/workflows/publish_browserstack.yml:1]: Yes, I follow a pattern in other repositories we have, to have a clear indication for Tal for example that this action could be run manually. I don't like how it looks when it was run automatically, but I don't see a better solution.

**Pull request merged**